homeassistant/helpers/template.py

```diff
@@ -1776,6 +1776,11 @@ def regex_match(value, find="", ignorecase=False):
     return bool(_regex_cache(find, flags).match(value))
 
 
+def contains(seq, value):
+    """Opposite of the ``in`` test, allowing use as a test in filters like ``selectattr``."""
+    return value in seq
```

Comment:
I think [`operator.contains`](https://docs.python.org/3.11/library/operator.html#operator.contains) does the same thing...

Reply:
without checking much more into it, i agree that it look like it work the same.

But it's an extra library to import, for something really basic, and i am of the opinion that in the jinja world, most of the extra filter people use should be iso.

Ansible is one of the biggest jinja based project, so i feel like keeping the same code is the way to go about it.

What's do you think ?

Reply:
`operator` is a built-in library.

The point is you wouldn't need to define this function at all; just `from operator import contains`

Reply:
You are right, and the test don't move.
I'm not a fan personally but i am also not a dev

Your change have been committed, if this is validated i'll squash it.